plugins/modules/rabbitmq_user.py

```diff
@@ -77,6 +77,14 @@
       - This option will be ignored when permissions option is used.
     type: str
     default: '^$'
+  topic_permissions:
+    description:
+      - A list of dicts, each dict contains vhost, exchange, read_priv and write_priv,
+        and represents a topic permission rule for that vhost.
+      - By default vhost is / and exchange is amq.topic.
```

Comment:
```suggestion
      - By default vhost is C(/) and exchange is C(amq.topic).
```

Reply:
I'd also put a note here that the feature `has been added in RabbitMQ 3.7.0.`

Reply:
or "supported since ..."

Reply:
and that the option is ignored otherwise / the module will fail. Though I didn't look at the code so far, so I have no idea how it is implemented.

Reply:
@abompard ^